examples/mysql/mysql_status.rb

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env ruby
+
+# Ensure your instrument_server process runs with these environment variables set,
+# or replace with your own.
+
+MYSQL_HOST              = ENV["MYSQL_HOST"]
+MYSQL_PORT              = ENV["MYSQL_PORT"] || 3306
+MYSQL_USER              = ENV["MYSQL_USER"]
+MYSQL_DEFAULTS_FILE     = ENV["MYSQL_DEFAULTS_FILE"]
+MYSQL_PASSWORD          = ENV["MYSQL_PASSWORD"]
+
+RATE_METRICS_TO_INSPECT = %w{Queries Bytes_sent Bytes_received Connections Slow_queries}
```

Comment:
can we downcase these guys in the actual metric names we return to instrumental?